packages/core/src/components/my-container/my-container.tsx

```diff
@@ -0,0 +1,80 @@
+import { Component, Method, State, h } from '@stencil/core';
```

Comment:
my-container is only being used to show how w2w-select works. Can be removed in the future